Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            PersonInfoAskerAndPrinter();
+            PersonInfoAskerAndPrinter();
+        }
+        private static void PersonInfoAskerAndPrinter()
```

Comment:
This also needs a change.
To explain it for the last comment and for this one:

If you would imagine something when reading "Asker" or "Printer" you would think of some object or being that performs this task. A **printer** IRL **prints** documents that you want to be printed. Note the difference!

A good name could be "PromptPersonData()" or "ProcessPerson()" or something like that. Feel free to ask again if you need more details on that subject. :)

Reply:
Oh thank you so much i missed that one.
thanks for explaining as wel very helpful.